feature-libs/cart/quick-order/components/quick-order/quick-order.component.html

```diff
@@ -1,37 +1,49 @@
-<div class="quick-order-header">
-  <h3>{{ 'quickOrderList.addProducts' | cxTranslate }}</h3>
-</div>
+<ng-container *ngIf="quickOrderListLimit$ | async as quickOrderListLimit">
+  <ng-container *ngIf="entries$ | async as entries">
+    <div class="quick-order-header">
+      <h3>{{ 'quickOrderList.header' | cxTranslate }}</h3>
+      <p>
+        {{
+          'quickOrderList.subHeader'
+            | cxTranslate: { limit: quickOrderListLimit }
+        }}
+      </p>
+    </div>
 
-<div class="quick-order-form-body">
-  <cx-quick-order-form></cx-quick-order-form>
-</div>
+    <div class="quick-order-form-body">
+      <cx-quick-order-form
+        [isDisabled]="entries.length >= quickOrderListLimit"
+      ></cx-quick-order-form>
+    </div>
 
-<div class="quick-order-table-body">
-  <cx-quick-order-table></cx-quick-order-table>
-</div>
+    <div class="quick-order-table-body">
+      <cx-quick-order-table></cx-quick-order-table>
```

Comment:
Just leaving a note so that you can ask the reviewer for this template before on why don't you pass the entries as input to the QuickOrderTableComnent as you are calling the entries from there as well too.